conjure-java-core/src/test/java/com/palantir/conjure/java/EteBinaryResource.java

```diff
@@ -32,6 +38,20 @@ public StreamingOutput postBinary(AuthHeader _authHeader, InputStream body) {
         return output -> output.write(ByteStreams.toByteArray(body));
     }
 
+    @Override
+    public StreamingOutput postBinaryThrows(
+            @NotNull AuthHeader authHeader, int bytesToRead, @NotNull InputStream body) {
+        if (bytesToRead > 0) {
+            try {
+                ByteStreams.copy(ByteStreams.limit(body, bytesToRead), ByteStreams.nullOutputStream());
```

Comment:
Is this just the same as `ByteStreams.exhaust(ByteStreams.limit())` ?

Reply:
Yep, I wasn't aware of exhaust.